cdp_scrapers/legistar_utils.py

```diff
@@ -515,15 +515,10 @@ def get_legistar_content_uris(client: str, legistar_ev: dict) -> ContentUriScrap
     # return false;"
     # href="#" style="color:Blue;font-family:Tahoma;font-size:10pt;">Video</a>
     extract_url = soup.find(
-        "a",
-        id=re.compile(r"ct\S*_ContentPlaceHolder\S*_hypVideo"),
-        class_="videolink",
+        "a", id=re.compile(r"ct\S*_ContentPlaceHolder\S*_hypVideo"), onclick=True
```

Comment:
Going by the PR description, that the `videolink` class filtering is too strict, I agree it should be OK to remove it. But adding this new criteria for the onclick attribute is incorrect, because we will return `UnrecognizedPatternError` when this result is `None`. That exception is meant to tell us that something has changed on these web pages.

You have the check for this onclick attribute below, where you correctly return `ContentNotProvidedError` to say video is not available.

i.e. I think we simply need to remove this `onclick=True` query term. I tested locally by checking out this branch, merging `upstream/main` then removing this onclick filtering. The test failed but because my connection got throttled and was `forcibly reset by peer`. :sweat_smile: